js/apps/keycloak-server/scripts/start-server.js

```diff
@@ -50,19 +61,43 @@ async function startServer() {
   await importClient();
 }
 
-async function downloadServer() {
+function handleArgs(args) {
+  const { values, tokens } = parseArgs({
+    args,
+    options,
+    strict: false,
+    tokens: true,
+  });
+  // we need to remove the args that belong to the script so that we can pass the rest through to keycloak
+  tokens
+    .filter((token) => options.hasOwn(token.name))
+    .forEach((token) => {
+      let tokenRaw = token.rawName;
+      if (token.value) {
+        tokenRaw += `=${token.value}`;
+      }
+      args.splice(args.indexOf(tokenRaw), 1);
+    });
+  return { values, args };
+}
+
+async function downloadServer(local) {
   const directoryExists = fs.existsSync(SERVER_DIR);
 
   if (directoryExists) {
     console.info("Server installation found, skipping download.");
     return;
   }
 
-  console.info("Downloading and extracting server…");
-
-  const nightlyAsset = await getNightlyAsset();
-  const assetStream = await getAssetAsStream(nightlyAsset);
-
+  var assetStream;
+  if (local) {
+    console.info("Looking for " + LOCAL_DIST_NAME + " at " + LOCAL_QUARKUS);
```

Comment:
Let's use a template literal here.
```suggestion
    console.info(`Looking for ${LOCAL_DIST_NAME} in ${LOCAL_QUARKUS}`);
```